src/go/rpk/pkg/cli/cmd/acl/user.go

```diff
@@ -178,29 +177,20 @@ func printUsernames(usernames []string) {
 }
 
 func buildAdminAPI(
-	apiUrls *[]string, tls func() (*config.TLS, error),
+	apiUrls *[]string, tls func() (*tls.Config, error),
 ) func() (admin.AdminAPI, error) {
 	return func() (admin.AdminAPI, error) {
 		if len(*apiUrls) == 0 {
 			return nil, errors.New("--api-urls is required")
 		}
-		tlsFiles, err := tls()
+		tlsConfig, err := tls()
 		if err != nil {
 			return nil, err
 		}
-		if tlsFiles == nil {
+		if tlsConfig == nil {
```

Comment:
This check is now no longer necessary and these three lines can be eliminated.

Reply:
Good catch.